spec/index.md

```diff
@@ -3193,7 +3234,8 @@ is_effective_canister_id(E.content, ECID)
       S.freezing_threshold[E.content.canister_id],
       memory_usage_wasm_state(S.canisters[E.content.canister_id].wasm_state) +
         memory_usage_raw_module(S.canisters[E.content.canister_id].raw_module) +
-        memory_usage_canister_history(S.canister_history[E.content.canister_id]),
+        memory_usage_canister_history(S.canister_history[E.content.canister_id]) +
+        memory_usage_snapshot(S.snapshots[E.content.canister_id]),
```

Comment:
S.snapshots should be introduced (along with the type) in the definition of the IC state (abbreviated by S here) and initiliazed in the definition of the initial IC state.

Reply:
I am not sure where exactly you mean. You mean here: https://github.com/dfinity/interface-spec/blob/master/spec/index.md?plain=1#L2960?

Reply:
yes and also here: https://github.com/dfinity/interface-spec/blob/125ca124ddfcfcda520ca107e266e8faaeb627bc/spec/index.md?plain=1#L3028